getting-started/installation-timescale-cloud.md

```diff
@@ -0,0 +1,25 @@
+## Installation (Timescale Cloud) [](installation-timescale-cloud)
```

Comment:
Why is this stuff in this PR?

Reply:
No idea. Will check.

Reply:
Couldn't see it in the diff I just pushed. Will check the newly pushed one.